### PR TITLE
Workspace overview layout tweaks

### PR DESCRIPTION
**Pull request by jakolehm**
Targets #2047

https://user-images.githubusercontent.com/1446224/110467530-cff60000-80df-11eb-9cd3-06f2203365a5.mp4

### CODE REVIEW

**Comment by jakolehm** [comment]
We need to figure out how to fix integration tests ...

File: src/renderer/components/cluster-manager/clusters-menu.tsx [hunk 1]
```diff
@@ -149,14 +144,25 @@ export class ClustersMenu extends React.Component<Props> {
...
-        <div className="add-cluster">
-          <Tooltip targetId="add-cluster-icon">
-            Add Cluster
-          </Tooltip>
-          <Icon big material="add" id="add-cluster-icon" disabled={workspace.isManaged} onClick={this.addCluster}/>
-          {newContexts.size > 0 && (
-            <Badge className="counter" label={newContexts.size} tooltip="new"/>
-          )}
+
+        <div className="WorkspaceMenu">
```
Comment (jim-docker) [inline src/renderer/components/cluster-manager/clusters-menu.tsx:148]: would seem more natural to put this at the top, above the cluster icons

File: src/renderer/components/cluster-manager/clusters-menu.tsx [hunk 2]
```diff
@@ -149,14 +144,25 @@ export class ClustersMenu extends React.Component<Props> {
...
+            close={() => this.workspaceMenuVisible = false}
+            toggleEvent="click"
+          >
+            <MenuItem onClick={() => navigate(addClusterURL())} data-test-id="add-cluster-menu-item">
+              <Icon small material="add" /> Add Cluster
+            </MenuItem>
+            <MenuItem onClick={() => navigate(landingURL())} data-test-id="workspace-overview-menu-item">
+              <Icon small material="dashboard" /> Workspace Overview
+            </MenuItem>
+          </Menu>
```
Comment (jim-docker) [inline src/renderer/components/cluster-manager/clusters-menu.tsx:165]: this stays visible when user clicks on nav sidebar/dashboard, even after clicking "open cluster settings" icon in top right corner. I haven't figured out how to fix it.

File: src/renderer/components/+landing-page/workspace-cluster.store.ts [hunk 3]
```diff
@@ -6,23 +6,35 @@ import { autobind } from "../../utils";
...
 export class ClusterItem implements ItemObject {
   constructor(public cluster: Cluster) {}
-  
+
   get name() {
     return this.cluster.name;
   }
 
+  get distribution() {
+    return (this.cluster.metadata["distribution"] || "unknown").toString();
```
Comment (Nokel81) [inline src/renderer/components/+landing-page/workspace-cluster.store.ts:15]:
```suggestion
    return this.cluster.metadata?.distribution?.toString() ?? "unknown";
```

File: src/renderer/components/cluster-manager/clusters-menu.tsx (hunk 1 above)
Comment (Nokel81) [reply src/renderer/components/cluster-manager/clusters-menu.tsx:148]: IMO that sounds like a good idea.

Comment (jakolehm) [reply src/renderer/components/cluster-manager/clusters-menu.tsx:148]: Just a note that the placement of this icon was already agreed with @miskun .

**Comment by Nokel81** [comment]
I cannot seem to find it in this PR (or its target) but the fact that the blue "Add" button is not in the corner exactly catches my eye very easily.

**Comment by jakolehm** [comment]
> I cannot seem to find it in this PR (or its target) but the fact that the blue "Add" button is not in the corner exactly catches my eye very easily.

Placement of that button is out-of-scope of this PR. `ItemListLayout` is responsible for drawing this, see Lens 4.1.4 screenshot as an example: 
image

**Pull request merged**